internal/pkg/template/input/validation.go

```diff
@@ -136,17 +143,29 @@ func validateUserInputByReflectKind(userInput interface{}, expectedType reflect.
 
 	if reflect.TypeOf(userInput).Kind() != expectedType {
 		return fmt.Errorf(
-			"the input should be a type %s, got %s instead",
+			"%s should have type %s, got %s instead",
+			fieldName,
 			expectedType.String(),
 			reflect.TypeOf(userInput).Kind().String(),
 		)
 	}
 	return nil
 }
 
-func validateUserInputWithRules(userInput interface{}, rules string, ctx context.Context) error {
+func validateUserInputWithRules(userInput interface{}, rules string, ctx context.Context, fieldName string) error {
 	validate := goValidator.New()
-	return validate.VarCtx(ctx, userInput, rules)
+	enLocale := en.New()
+	universalTranslator := ut.New(enLocale, enLocale)
+	enTranslator, found := universalTranslator.GetTranslator("en")
+	if !found {
+		panic(fmt.Errorf("en translator was not found"))
+	}
+	err := enTranslation.RegisterDefaultTranslations(validate, enTranslator)
+	if err != nil {
+		panic(fmt.Errorf("translator was not registered: %w", err))
+	}
+
+	return translateError(validate.VarCtx(ctx, userInput, rules), enTranslator, fieldName)
```

Comment:
Myslim, ze tu by sa finalne mohol volat nas balicek `validator.Validate`, ctx parameter sa tam moze doplnit.
A volalo by sa tam `VarCtx` namiesto `Var`. Kde nas context nezaujima by sa dalo `context.Background()`.
Co mysli?

A tam v `validator.Validate` by bol zaregistrovany aj translator.
Mozno to tak mas v plane, len poznamka.

Reply:
Miesta, kde by bolo potrebne upravit existujuce volania `validator.Validate` t(ak pridas ctx) ti pekne vypise `make lint`.

Reply:
Jo, to jsem se chtěl vlastně zeptat, jestli by se měl ten translation přidat už do `validator.Validate`, a mám v plánu to zobecnit ale až teď budu řešit ty chybový hlášky pro validaci definice. Ale nic nebrání tomu to udělat hned. 🙂

Reply:
ok 🙂